app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,50 @@
+class FoodsController < ApplicationController
+  # before_action :set_food, only: %i[new show destroy]
+
+  # GET /foods or /foods.json
+  def index
+    # @user = current_user
+    # @foods = Food.where(user_id: @user.id)
```

Comment:
- [ ] let's remove commended code as it makes code hard to read.

Reply:
Following commit for the solution:

https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/cb40a1da187ce86386e47dd0e40c0cf3546eeed9